src/Driver/Driver.jl

```diff
@@ -25,7 +25,7 @@ Base.@kwdef mutable struct CLIMA_Settings
     mpi_knows_cuda::Bool = false
     show_updates::Bool = true
     update_interval::Int = 60
-    enable_diagnostics::Bool = true
+    enable_diagnostics::Bool = false
```

Comment:
Unfortunately this doesn't actually change the default. I thought I'd added a comment there to explain this, but clearly not. I'll fix this in #810 if you're decided that diagnostics should not be enabled by default?

Reply:
I am fine either way :).